src/calendarmanager.cpp

```diff
@@ -224,12 +224,13 @@ QString NemoCalendarManager::getNotebookColor(const QString &notebookUid) const
 
 void NemoCalendarManager::cancelAgendaRefresh(NemoCalendarAgendaModel *model)
 {
-    mAgendaRefreshList.removeOne(model);
+    if (!mAgendaRefreshList.isEmpty())
+        mAgendaRefreshList.removeOne(model);
```

Comment:
Should this second commit have any effect? Removing not found item should already just return false and empty list will not contain any value.